CHANGELOG.md

```diff
@@ -27,8 +27,8 @@
 
 ## Fixed Issues
 
-- [Generator] Fix the function import (OData V2 + V4) and action import, where the return type is an primitive edm type like `Edm.String`.
-- [CSRF] Fix the error message of fetching the csrf token, it should show original error.
```

Comment:
So, you think this is not relevant topic for the user?

Reply:
Seems like implementation details to me. Would you prefer to keep it?

Reply:
Since recently, some batch/csrf token released issue were created, I would like to avoid the confusing when people don't see the error message. With the change log, I can tell from this version, you can see more details.

Reply:
Ok,  I added it in the improvements section 😊